README.md

```diff
@@ -65,14 +65,14 @@ Create a unit file from parameters
 
 ```puppet
 systemd::manage_unit { 'myrunner.service':
-  $unit_entry    => {
+  unit_entry    => {
     'Description' => 'My great service',
   },
-  $service_entry => {
+  service_entry => {
     'Type'       => 'oneshot',
     'ExecStart' => '/usr/bin/doit.sh',
   },
-  $install_entry => {
+  install_entry => {
     'WantedBy' => 'multi-user.target',
   },
   enable         => true,
```

Comment:
Oh, I just noticed: could you align `=>` on this and the line below?

Reply:
sure, there you go